src/compiler/transformers/add-static-style.ts

```diff
@@ -24,15 +30,15 @@ export const addStaticStyleGetterWithinClass = (
 
 /**
  * Adds static "style" property to the class variable.
+ * ```typescript
  * const MyComponent = class {}
  * MyComponent.style = "styles";
+ * ```
+ * @param styleStatements a list of statements containing style assignments to a class
+ * @param cmp the metadata associated with the component being evaluated
  */
-export const addStaticStylePropertyToClass = (
-  styleStatements: ts.Statement[],
-  cmp: d.ComponentCompilerMeta,
-  commentOriginalSelector: boolean
```

Comment:
In searching the code, this was only called in one place, with a value of `false`. I can revert this if we want, but it seemed like an easy win once I dug through the calls that we were making here

Reply:
nah that sounds good :) I lean on the side of YAGNI w/ such things